pkg/relayer/proxy/websockets/connection.go

```diff
@@ -81,9 +81,9 @@ type connection struct {
 	isClosed atomic.Bool
 }
 
-// connectServiceBackend establishes a websocket connection established by the
+// ConnectServiceBackend establishes a websocket connection established by the
 // relay miner client to the service backend endpoint.
-func connectServiceBackend(serviceBackendUrl *url.URL, header http.Header) (*websocket.Conn, error) {
+func ConnectServiceBackend(serviceBackendUrl *url.URL, header http.Header) (*websocket.Conn, error) {
```

Comment:
Why did you make this public?

Reply:
I made this function public so I can reuse it in the `relayMinerHTTPServer.forwardAsyncConnection()` method (`pkg/relayer/proxy/async.go`). I wanted the forward server to behave the same way as the relay server, so I reused the same WebSocket dialer configuration.